src/robot_dart/graphics/graphics.hpp

```diff
@@ -80,7 +80,15 @@ namespace robot_dart {
                     osg::Vec3d(_camera_pos(0), _camera_pos(1), _camera_pos(2)), osg::Vec3d(_look_at(0), _look_at(1), _look_at(2)), osg::Vec3d(_camera_up(0), _camera_up(1), _camera_up(2)));
                 _osg_viewer->home();
             }
-
+	  
+	    void add_grid(double size = 1, size_t number = 10)
+	    {
+	        osg::ref_ptr<dart::gui::osg::GridVisual> grid = new dart::gui::osg::GridVisual();
```

Comment:
Thank you for this PR! Are we sure that every DART version >= 6 (that we support with robot_dart) has `osg::GridVisual`? I have a feeling that this was introduced quite lately (I might be wrong)..